scripts/setup-kustomize.sh

```diff
@@ -130,8 +141,32 @@ if [ -n "$PRIVATE_REG_SERVER" ]; then echo "YES"; else echo "NO"; fi
 echo -n "Add server mounts: "
 if [ -n "$USE_SERVER_MOUNT_PATCH" ]; then echo "YES"; else echo "NO"; fi
 echo "Deployment method: $VERTICA_DEPLOYMENT_METHOD"
+echo "NMA running mode: $NMA_RUNNING_MODE"
 echo "Vertica superuser name: $VERTICA_SUPERUSER_NAME"
 
+if [ "$NMA_RUNNING_MODE" == "sidecar" ]
+then
+    cp $RESTART_SANITY/sidecar-40-assert.yaml $RESTART_SANITY/40-assert.yaml
```

Comment:
I know this does not look good but kuttl does not support partial comparison of slices in asserts so I did not have many options

Reply:
I agree this is ugly. What if added a feature flag in the operator via an annotation that let you control the index of the server container. Then for the tests that have a strict dependency that server is first we use that feature flag. This flag would only be used in test purposes and won't be externalized.